### ISSUE TITLE
Support configuration for author and contact information

### ISSUE THREAD

**Issue by brnstz** [brnstz/md2shunn#1]

**Comment by urmaul** [comment]
I have added configuration via YAML front matter in [my fork](https://github.com/urmaul/md2shunn). Should I try to merge it upstream?

**Comment by brnstz** [comment]
I took a look at your fork. It looks like this configuration goes into the .md file itself? I'd prefer having this as a global config (e.g., like a `.gitconfig` file).

**Comment by urmaul** [comment]
Yeah, it's not documented well... You can have configuration in the .md file itself (works well for title) but you can also have another .md file with global configuration. Configs from these two files will be merged.

And I couldn't come up with nice location for configuration file so I accept it as another CLI parameter.
